hedera/requirements/base.in

```diff
@@ -25,4 +25,4 @@ tqdm~=4.63.0
 # 	https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
 # 	https://github.com/protocolbuffers/protobuf/issues/10051
 protobuf==3.20.1
-pandas~=1.3.5
\ No newline at end of file
+pandas~=1.4.4
```

Comment:
Note that we could probably bump this to the latest 1.x release `1.5.3` without any issues. I briefly glanced at the [Pandas Release Notes](https://pandas.pydata.org/docs/whatsnew/) and didn't see any major incompatibilities from 1.4 to 1.5. Not sure about the jump to 2.0 though. AFAIK pandas is only being used in the import scripts.

Reply:
Sounds good will do!